orbit-2-core/orbit-2-core_build.gradle.kts

```diff
@@ -26,9 +26,9 @@ dependencies {
     compileOnly(ProjectDependencies.androidxAnnotation)
 
     // Testing
-    GroupedDependencies.testsImplementationJUnit5.forEach { testImplementation(it) }
-    testRuntimeOnly(ProjectDependencies.junitJupiterEngine)
     testImplementation(project(":orbit-2-test"))
+    GroupedDependencies.testsImplementation.forEach { testImplementation(it) }
+    testRuntimeOnly(ProjectDependencies.junitJupiterEngine)
```

Comment:
Since we're doing this can we get rid of `GroupedDependencies.testsRuntime` usages elsewhere?

Reply:
copy paste error here... but yes that would make sense